Sources/ScipioKit/Runner.swift

```diff
@@ -219,6 +219,7 @@ extension Runner {
             case disabled
             case project
             case storage(any CacheStorage, Set<CacheActorKind>)
```

Comment:
How about using `static let` instead of case to reduce duplicated cases?

```swift
enum CacheMode {
  static let disabled: Self = .storages([])
  static func storage(storage: any CacheStorage, actor: Set<CacheActorKind>) -> Self {
      .storages([(storage, actor)])
  }
}
```

Reply:
I'll try it.